examples/plotting/file/toolbar_autohide.py

```diff
@@ -3,7 +3,7 @@
 
 .. bokeh-example-metadata::
     :apis: bokeh.plotting.figure.scatter, bokeh.models.tools.Toolbar.autohide
-    :refs: :ref:`reference` > :ref:`reference_models` > :ref:'reference_models_tools` > :ref:'reference_models_tools_Toolbar` > :ref:'reference_models_tools_Toolbar_autohide`, :ref:`first_steps` > :ref:`first_steps_first_steps_4` > `first_steps_first_steps_4_deactivating_and_hiding_the_toolbar`# noqa: E501
+    :refs: :ref:`first_steps_4_toolbar`, :ref:`bokeh.models.tools`
```

Comment:
Are the existing references out of date? Unless the more granular references simply don't exist, then we should retain the fine-grained section links.